components/function-controller/internal/webhook/defaulting_webhook_test.go

```diff
@@ -126,7 +126,7 @@ func TestDefaultingWebHook_Handle(t *testing.T) {
 				// add /spec/scaleConfig
 				// add /status
 				// add /metadata/creationTimestamp
-				operationsCount: 6,
+				operationsCount: 4,
```

Comment:
You should update the comment above.

The other thing that this assertion is not good and we should think about checking the operations, not only the amount of them.

Reply:
Updated

Reply:
I created issue for this test: https://github.com/kyma-project/kyma/issues/15488